go/vt/vtctl/vtctl.go

```diff
@@ -1392,6 +1392,7 @@ func commandExecuteFetchAsApp(ctx context.Context, wr *wrangler.Wrangler, subFla
 func commandExecuteFetchAsDba(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
 	maxRows := subFlags.Int("max_rows", 10000, "Specifies the maximum number of rows to allow in fetch")
 	disableBinlogs := subFlags.Bool("disable_binlogs", false, "Disables writing to binlogs during the query")
+	disableForeignKeyChecks := subFlags.Bool("disable_foreign_key_checks", false, "toggles foreign_key_checks=0 before the query and =1 after. NOTE: this flag has no effect on tablets <= 12.0")
```

Comment:
Is it possible for `foreign_key_checks` to be already `0` and we change it to `1` if the user mistakenly supplies this flag?